tests/suites/test_suite_pkcs5.data

```diff
@@ -142,6 +142,30 @@ PBES2 Decrypt (Invalid padding & PKCS7 padding enabled)
 depends_on:MBEDTLS_MD_CAN_SHA1:MBEDTLS_DES_C:MBEDTLS_CIPHER_MODE_CBC:MBEDTLS_CIPHER_PADDING_PKCS7
 pbes2_decrypt:MBEDTLS_ASN1_CONSTRUCTED | MBEDTLS_ASN1_SEQUENCE:"301B06092A864886F70D01050C300E04082ED7F24A1D516DD702020800301406082A864886F70D030704088A4FCC9DCC394910":"70617373776f7264":"1B60098D4834CA752D37B430E70B7A085CFF86E21F4849F969DD1DF623342662443F8BD1252BF83CEF6917551B08EF55A69C8F2BFFC93BCB2DFE2E354DA28F896D1BD1BFB972A1251219A6EC7183B0A4CF2C4998449ED786CAE2138437289EB2203974000C38619DA57A4E685D29649284602BD1806131772DA11A682674DC22B2CF109128DDB7FDA3488A7144097565":144:MBEDTLS_ERR_PKCS5_PASSWORD_MISMATCH:"308187020100301306072A8648CE3D020106082A8648CE3D030107046D306B0201010420F12A1320760270A83CBFFD53F6031EF76A5D86C8A204F2C30CA9EBF51F0F0EA7A1440342000437CC56D976091E5A723EC7592DFF206EEE7CF9069174D0AD14B5F768225962924EE500D82311FFEA2FD2345D5D16BD8A88C26B770D55CD8A2A0EFA01C8B4EDFF060606060607"
 
+PBES2 Encrypt AES-128-CBC (OK)
```

Comment:
Minor: the provenance of the test data is documented in the commit message, but it would probably be good to have at least a hint directly in the file too. For example add "OpenSSL" in each test's description, and/or add a comment to the first test of the series explaining how the data was generated.

(Note: in the past we had lots of test data of undocumented provenance. Documenting it in the commit message is already good, but I'm trying to raise the bar even further.)

Reply:
Counterpoint: a comment could get out of date if the test data changes, a commit message doesn't. So I prefer the commit message for documenting how the data was generated. A comment is useful for a slightly different reason: explaining how to generate similar data (for example, to generate a non-regression test case for a bug fix in a corner case).

Reply:
> Counterpoint: a comment could get out of date if the test data changes, a commit message doesn't.

Good point, agreed. I'd still like something like "(OpenSSL)" in the test case description as a hint that it wasn't taken from an RFC or other standard source of known-answers tests, but I won't insist on it.